server/i-cal-feed.js

```diff
@@ -0,0 +1,52 @@
+import moment from "moment";
+import ics from "ics";
+
+const iCalFeed = {
+  handleRequest(params, req, res) {
+    console.log("iCalFeed.handleRequest called");
+
+    const hangouts = Hangouts.find({
+      end: {
+        $gte: moment()
+          .subtract(1, "months")
```

Comment:
@gauravchl can we make this configurable via the meteor settings or env var so we can easily adjust if we choose to in the future?

Reply:
@d3vild06 Can you elaborate on what we should put into meteor settings or env variables to more easily adjust, Roberto? I'm a little confused, because we don't have any API keys here; going to codebuddies.org/i-cal-feed will download the calendar!

Reply:
I was referring to the month interval. Instead of hard coding "month" or "1" can we use meteor settings to set them instead.

Reply:
Got it! Do you think we'll need to change that setting often? I personally don't care too much to see hangouts that happened over a month ago, although it might be interesting to toggle back. 

Although... Meteor settings are a pain to manage since I have to manually add it to a file in the droplet and do a bunch of server restarts, so if we can get away without not putting it in there, then I'm fine with that. :)